packages/web-app-files/src/services/archiver.ts

```diff
@@ -111,6 +132,12 @@ export class ArchiverService {
     }
     return urlJoin(configurationManager.serverUrl, this.capability.archiver_url)
   }
+
+  private getFileNameFromResponseHeaders(headers) {
+    const contentDispositionArr = headers['content-disposition']?.split('"')
+    const fileName = contentDispositionArr[1] ?? null
+    return fileName ? decodeURI(fileName) : 'download.tar'
```

Comment:
Are there edge cases where the archiver service doesn't give a file name? in that case `download.tar` would be wrong on a windows machine 🙈

Reply:
ftr not blocking the merge.

Reply:
> Are there edge cases where the archiver service doesn't give a file name?

Both serves deliver the file name via `content-disposition`. So I guess we just remove the default if it doesn't make sense anyway.